modules/local/get_chrom_sizes.nf

```diff
@@ -0,0 +1,35 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options = [:]
+
+/*
+ * Get chromosome sizes from a fasta file
+ */
+process GET_CHROM_SIZES {
+//    publishDir "${params.outdir}",
+//        mode: params.publish_dir_mode,
+//        saveAs: { filename -> saveFiles(filename:filename, publish_dir:"genome") }
+
+    conda     (params.enable_conda ? "bioconda::samtools=1.10" : null)
+    container "quay.io/biocontainers/samtools:1.10--h9402c20_2"
```

Comment:
Just noticed: does this quay link also support Singularity? Asking given the nf-core/modules always have two links - one from quay (docker) and the other from galaxy (singularity).

Sorry if this has been discussed before 😅

Reply:
I see other local modules have such options along the same lines (e.g. get_nanolyse_fasta)

Reply:
I have updated this to the following:
```
    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
        container "https://depot.galaxyproject.org/singularity/samtools:1.13--h8c37831_0"
    } else {
        container "quay.io/biocontainers/samtools:1.13--h8c37831_0"
    }
```